db/models/mining-monthly-report.js

```diff
@@ -222,6 +205,18 @@ module.exports = (sequelize, DataTypes) => {
         apps[app.id] = app;
         return -app.memoryRanking;
       });
+      let remainingBTC = btcPayoutTotal;
+      sorted = sorted.map((app) => {
+        const btcPayout = remainingBTC * btcPayoutDecay;
```

Comment:
We're good with any imprecision introduced with a calculation like this?

Reply:
Yes - this is only for display purposes, and is not used for actual accounting. Although, it wouldn't hurt to refactor at some point. This whole file is a bit of a mess and needs unit tests.